test/spec/behavior/generated-parser-behavior.spec.js

```diff
@@ -1194,19 +1194,33 @@ describe( "generated parser behavior", function () {
 
                 } );
 
-                it( "discards any expectations recorded when matching the expression", function () {
+                it( "not discards any expectations recorded when matching the expression", function () {
 
```

Comment:
this would make more sense: "doesn't discard any expectations recorded when matching the expression"